server/_lib/parser.ts

```diff
@@ -12,7 +12,15 @@ export const constructHtml = (data: TweetSyndication, options: TweetOptions, isQ
     const tweet_class = isQuotedTweet
       ? mapClassOptions("tweet").replace("w-[400px]", "").replace("w-[500px]", "").concat(" mt-4")
       : mapClassOptions("tweet")
-
+    
+    let favorite_count_str;
+    if (favorite_count >= 1000000){
+        favorite_count_str = (favorite_count/1000000).toFixed(1)+' m';
+    } else if (favorite_count >= 1000){
+        favorite_count_str = (favorite_count/1000).toFixed(1)+' K';
```

Comment:
I think we can change this conditional to render `(favorite_count/1000).toFixed(1)+' K';` when it is >= 10000, and
add a comma (eg: 4678 -> 4,678 ) for >= 1000.  eg: https://twitter.com/9GAG/status/1568812082140684289